lib/src/components/NxRadio/NxRadio.scss

```diff
@@ -10,52 +10,78 @@
 @import '../../scss-shared/nx-input-helpers';
 
 .nx-radio {
-  $circle-width: 16px;
-  $label-margin: 8px;
-  $line-height: 20px;
-
   @include container-horizontal;
   @include regular();
   @include focusable-toggle-input(".nx-radio__outer-circle");
 
-  display: inline-block;
-  font-size: 14px;
-  line-height: $line-height;
+  display: block;
+  margin-bottom: $nx-spacing-xs;
   position: relative;
   white-space: nowrap;
+  max-width: 100px;
```

Comment:
Did you mean `min-width`?
![image](https://user-images.githubusercontent.com/3630091/87595537-b28da500-c6bc-11ea-8413-aa0c7043cc5b.png)

Reply:
> Did you mean `min-width`?
> ![image](https://user-images.githubusercontent.com/3630091/87595537-b28da500-c6bc-11ea-8413-aa0c7043cc5b.png)

No, if it's `min-width` then the element will still expand to full width which is what I'm trying to avoid. What you see above in your screenshot is expected. Either `width` or `max-width` give the results we want.

Reply:
> > Did you mean `min-width`?
> > ![image](https://user-images.githubusercontent.com/3630091/87595537-b28da500-c6bc-11ea-8413-aa0c7043cc5b.png)
> 
> No, if it's `min-width` then the element will still expand to full width which is what I'm trying to avoid. What you see above in your screenshot is expected. Either `width` or `max-width` give the results we want.

Note that despite the fact that the `max-width` is only 100px the entire length of the label triggers the cursor change on hover, but _only_ the label. I have confirmed this behaviour in IE

Reply:
> Either width or max-width give the results we want.

So the result that you want is that if it's less than 100px wide you can click to the right of it, up to the 100px line, but if it's more than 100px wide you can only click on the text?

Reply:
While looking at this I found that it doesn't handle overflow properly.  In this case it appears to cut off at the edge of `<main>`

![image](https://user-images.githubusercontent.com/3630091/87798020-79b71280-c819-11ea-9a0b-f489483d7c11.png)

Reply:
> > Either width or max-width give the results we want.
> 
> So the result that you want is that if it's less than 100px wide you can click to the right of it, up to the 100px line, but if it's more than 100px wide you can only click on the text?

Yes

Reply:
> While looking at this I found that it doesn't handle overflow properly. In this case it appears to cut off at the edge of `<main>`
> 
> ![image](https://user-images.githubusercontent.com/3630091/87798020-79b71280-c819-11ea-9a0b-f489483d7c11.png)

After speaking with @dturner32 we decided on using ellipsis truncation on form labels, and so I have created a new variable for form label width. https://github.com/sonatype/sonatype-react-shared-components/pull/86/commits/223eb5b462605d3af1066750992960559edc03fe

Reply:
> > > Either width or max-width give the results we want.
> > 
> > 
> > So the result that you want is that if it's less than 100px wide you can click to the right of it, up to the 100px line, but if it's more than 100px wide you can only click on the text?
> 
> Yes

Solved this problem by using flex box for the layout: https://github.com/sonatype/sonatype-react-shared-components/pull/86/commits/ad7a2c1bab2dff105b449cbdd3309372d8bd5173

Reply:
> > > > Either width or max-width give the results we want.
> > > 
> > > 
> > > So the result that you want is that if it's less than 100px wide you can click to the right of it, up to the 100px line, but if it's more than 100px wide you can only click on the text?
> > 
> > 
> > Yes
> 
> Solved this problem by using flex box for the layout: [ad7a2c1](https://github.com/sonatype/sonatype-react-shared-components/commit/ad7a2c1bab2dff105b449cbdd3309372d8bd5173)

Didn't need the flex-shrink values: https://github.com/sonatype/sonatype-react-shared-components/pull/86/commits/ef865a5c5920b11d0b03f1253d58bcd12d2fc708

Reply:
> Didn't need the flex-shrink values: ef865a5

Doesn't look like the flex is taking effect to me: 
![image](https://user-images.githubusercontent.com/3630091/88070114-51445680-cb40-11ea-9e9f-b94dee5fb9db.png)

Reply:
> > Didn't need the flex-shrink values: [ef865a5](https://github.com/sonatype/sonatype-react-shared-components/commit/ef865a5c5920b11d0b03f1253d58bcd12d2fc708)
> 
> Doesn't look like the flex is taking effect to me:
> ![image](https://user-images.githubusercontent.com/3630091/88070114-51445680-cb40-11ea-9e9f-b94dee5fb9db.png)

Fixed here with `max-content`: https://github.com/sonatype/sonatype-react-shared-components/pull/86/commits/354c7030d195004b246fd5f513f6b0b8a78d1b59